### DISCUSSION TITLE
No modification of the website even though modifications as presented in the tutorial have been performed

### DISCUSSION THREAD

**Discussion by odileandres** [daattali/beautiful-jekyll#915] [Q&A]
Hello, 

I try to make a blog using a file .md.
I modify some values in the _config file, like the title and the image for the navbar-img to get an image for the head. Moreover, I removed the posts. However, nothing changes on the website, it's been more than 3 hours and I still have the original website. 

What should I do?

**Comment by daattali** [comment]
I don't see the default website. Here's what I see

![image](https://user-images.githubusercontent.com/952340/146184355-442d2473-f036-4e9d-ad1f-9a7cd7f51520.png)

**Comment by odileandres** [comment]
I solved the problem, thank you!

________________________________
De : Dean Attali ***@***.***>
Envoyé : mercredi 15 décembre 2021 13:11:28
À : daattali/beautiful-jekyll
Cc : Andres Odile Louise; Author
Objet : Re: [daattali/beautiful-jekyll] No modification of the website even though modifications as presented in the tutorial have been performed (Discussion #915)


I don't see the default website. Here's what I see

[image]<https://user-images.githubusercontent.com/952340/146184355-442d2473-f036-4e9d-ad1f-9a7cd7f51520.png>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/daattali/beautiful-jekyll/discussions/915#discussioncomment-1812278>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALQCTWMKNPBCGA5JXORYIV3URCAXBANCNFSM5KDNOCCQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
